lib/docs/scrapers/phoenix.rb

```diff
@@ -1,9 +1,8 @@
 module Docs
   class Phoenix < UrlScraper
     self.type = 'elixir'
-    self.release = '1.3.4'
     self.base_url = 'https://hexdocs.pm/'
-    self.root_path = 'phoenix/Phoenix.html'
+    self.root_path = "phoenix/#{release}/Phoenix.html"
```

Comment:
This isn't going to work because self.root_path is evaluated before you create the different releases, which means the root path will always be "phoenix//Phoenix.html".